apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -463,6 +473,21 @@ class SparseNDArrayRead(_SparseNDArrayReadBase):
         Maturing.
     """
 
+    def dask_array(
```

Comment:
this doesn't seem to support time travel. It probably should, as it would be unique in our API if it did not